docs/error-codes.md

```diff
@@ -1636,4 +1636,22 @@ void RequirePublicMethodOnAType(
       // can break functionality when embedded in a single-file app. Use 'MethodFriendlyToSingleFile' instead. http://help/assemblyfiles
       MethodWithAssemblyFilesUsage();
   }
+  ```
+
+#### `IL3003`: Presence of 'RequiresAssemblyFilesAttribute' on method 'method' doesn't match overridden method 'base method'. All overridden methods must have 'RequiresAssemblyFilesAttribute'.
```

Comment:
nit: Going forward, if we add any other `Requires...` attribute, we should re-use this diagnostic in the analyzer, so I think we should make it specific to `RequiresAssemblyFilesAttribute` here.